Riot/Modules/SetPinCode/SetPinCoordinatorBridgePresenter.swift

```diff
@@ -115,11 +123,14 @@ final class SetPinCoordinatorBridgePresenter: NSObject {
         }
     }
     
-    func dismiss() {
+    func dismissWithMainAppWindow(_ window: UIWindow) {
         guard let coordinator = self.coordinator else {
             return
         }
         coordinator.toPresentable().view.removeFromSuperview()
+        
+        window.makeKeyAndVisible()
```

Comment:
Is there a risk here that if we do not properly `dismiss` for every `present` (as opposed to navigate elsewhere) that the wrong window will be `keyAndVisible`?

Reply:
Yep, that's a possibility but the old pin screen presentation would've also blocked the whole app if not dismissed properly (albeit poorly sometimes). Given that we haven't seen any problems with that I think this is safe enough. I really want to avoid messing with existing code too much.